test/SlowTests/Issues/RavenDB_19548.cs

```diff
@@ -30,8 +30,8 @@ public async Task RawRavenQueryProjectionWithoutNewKeyword()
                 }
 
                 /* should work:
-                * simple linq select
-                */
+                 * simple linq select
```

Comment:
I don't think you should do any formatting changes in this file

Reply:
yes, I reverted the formatting of this file